deployment_scripts/linode-marketplace-openlitespeed-django/openlitespeed-django-deploy.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
```

Comment:
simliar to above comment, Is there a reason we took out the sudo user tasks?

Reply:
I was trying to make it replicate how it's currently set up. I'll add it back in, though, so we can stay consistent across the board, and also so we can save the creds to /home/$USER/.credentials